### PR TITLE
Fixed accessToken test, added core.services.messages.flatten (with test)

### PR DESCRIPTION
**Pull request by irjudson**
@timfpark  Can you merge this and publish a new module (if it passes CR); then I can finish testing flattened messages.

### CODE REVIEW

File: lib/services/messages.js [hunk 1]
```diff
@@ -318,6 +319,15 @@ var translate = function(message) {
     return message;
 };
 
+var flatten = function(message, opt) {
+    var options = {
+        safe : true
```
Comment (timfpark) [inline lib/services/messages.js]: What is the safe for here?

**Comment by timfpark** [comment]
LGTM

**Comment by timfpark** [comment]
PS.  You forgot to add 'flat' to package.json.  This is a good reason to always add packages to package.json and do an 'npm install' off of that vs. doing an 'npm install flat'

**Comment by irjudson** [comment]
I think I forgot to git add it 😬

**Pull request merged**